homeassistant/components/bbox/device_tracker.py

```diff
@@ -20,6 +20,9 @@
 from homeassistant.util import Throttle
 import homeassistant.util.dt as dt_util
 
+import urllib3
```

Comment:
We shouldn't make Home Assistant aware of these implementation details. These changes are more appropriate in [`pybbox`](https://github.com/HydrelioxGitHub/pybbox)

Reply:
Sorry, 

I didn't know, this is my first initiative.

I will look at the side of the bbox.py

Thanks, sorry again

Reply:
No need to apologize! 👍🏻